src/txgraph.cpp

```diff
@@ -3253,6 +3273,7 @@ std::vector<TxGraph::Ref*> TxGraphImpl::Trim() noexcept
                                 .subspan(group_data.m_cluster_offset, group_data.m_cluster_count);
         uint64_t size{0};
         for (Cluster* cluster : cluster_span) {
+            MakeAcceptable(*cluster, cluster->GetLevel(*this));
```

Comment:
28ff1be7b5cae0f0674e0d0045da0eee5e9adc6a

Could throw this on as extra belt and suspenders to ensure we didn't leave the cluster in a needs split state prior or something which would result in it not doing anything.

```Suggestion
            MakeAcceptable(*cluster, cluster->GetLevel(*this));
            Assume(cluster->IsAcceptable());
```